nio/events/room_events.py

```diff
@@ -1522,7 +1553,6 @@ class RoomUpgradeEvent(Event):
     Attributes:
         body (str): A server-defined message.
         replacement_room (str): The new room the client should be visiting.
-        content (dict): The content of the tombstone event.
```

Comment:
Why remove this?

Reply:
There is no class var called `content` in this class? I think the comment is wrong so I removed it.

Reply:
There's definitely a `content` present: https://spec.matrix.org/latest/client-server-api/#mroomtombstone

Reply:
I'll probably just modify your branch directly since honestly everything else LGTM 🙂